### PR TITLE
 TASK-50921 : Add the app into the app center

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: challenges-webapp/src/main/resources/locale/navigation/portal/global_en.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#
+# This file is part of the Meeds project (https://meeds.io/).
+# Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/resources/locale/navigation/portal/global_en.properties]:
```suggestion
# Copyright (C) 2021 Meeds Association
```

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/app-center-configuration.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/app-center-configuration.xml]:
```suggestion
Copyright (C) 2021 Meeds Association
```

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml]:
```suggestion
Copyright (C) 2021 Meeds Association
```

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/dynamic-container-configuration.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/dynamic-container-configuration.xml]:
```suggestion
Copyright (C) 2021 Meeds Association
```

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/portal-configuration.xml [hunk 5]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/portal-configuration.xml]:
```suggestion
Copyright (C) 2021 Meeds Association
```

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/portal/portal/global/navigation.xml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/portal/portal/global/navigation.xml]:
```suggestion
Copyright (C) 2021 Meeds Association
```

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/portal/portal/global/pages.xml [hunk 7]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
```
Comment (Rdidmohamedelhadi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/portal/portal/global/pages.xml]:
```suggestion
Copyright (C) 2021 Meeds Association
```

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

**Review by Rdidmohamedelhadi — DISMISSED** [review]
see that https://community.exoplatform.com/portal/dw/activity?id=370023
  **https://www.gnu.org/licenses/gpl-howto.en.html
  https://www.gnu.org/licenses/why-not-lgpl.en.html**

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+            <values-param>
+                <name>portal.resource.names</name>
+                <value>locale.navigation.portal</value>
+                <value>locale.portlet.Challenges</value>
+            </values-param>
+            <values-param>
+                <name>init.resources</name>
+                <value>locale.navigation.portal</value>
+                <value>locale.portlet.Challenges</value>
+
```
Comment (ahamdi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml]: Line to remove

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+        <component-plugin>
+        <name>Challenges Portlet Resource Bundle</name>
+        <set-method>addResourceBundle</set-method>
+        <type>org.exoplatform.services.resources.impl.BaseResourceBundlePlugin</type>
+        <init-params>
+          <values-param>
+            <name>classpath.resources</name>
+              <value>locale.navigation.portal</value>
+              <value>locale.portlet.Challenges</value>
+          </values-param>
```
Comment (ahamdi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml:32]: Fix indentation please

File: challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+            <values-param>
+                <name>portal.resource.names</name>
+                <value>locale.navigation.portal</value>
+                <value>locale.portlet.Challenges</value>
+            </values-param>
+            <values-param>
+                <name>init.resources</name>
+                <value>locale.navigation.portal</value>
+                <value>locale.portlet.Challenges</value>
```
Comment (ahamdi) [inline challenges-webapp/src/main/webapp/WEB-INF/conf/challenges/bundle-configuration.xml]: Is it needed to add both resource bundles in all bundle configurations

**Review by ahamdi — CHANGES_REQUESTED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**